designer/client/src/components/toolbars/scenarioActions/buttons/RunOffScheduleButton.tsx

```diff
@@ -34,7 +34,7 @@ export default function RunOffScheduleButton(props: ToolbarButtonProps) {
     const available = !disabled && isPossible && capabilities.deploy;
 
     const { open } = useWindows();
-    const action = (p, c) => HttpService.runOffSchedule(p, c).finally(() => dispatch(loadProcessState(processName, processVersionId)));
+    const action = (p, c, d) => HttpService.runOffSchedule(p, c).finally(() => dispatch(loadProcessState(processName, processVersionId)));
```

Comment:
Can we find a better name for these (p, c, d) parameters?

Reply:
Fixed, full parameter names.